docs-gen/content/rule_set/data_entry/data_types.md

```diff
@@ -57,6 +62,8 @@ The time and date format shall be as shown below, where the sub-second data and
 YYYY-MM-DDTHH:MM:SS.ssssssZ
 ```
 
+VSS also supports the unit `unix-time` that can be used together with datatype `uint32`/`uint64`/`int64`to indicate that the signal value correspond to number of non-leap seconds which have passed since 00:00:00 UTC on Thursday, 1 January 1970.
```

Comment:
Is this relevant here, "unix-time" being a `unit` (not a datatype), and we are also not explicitly talking about all the other units in the standard catalogue unit file.

Reply:
But with that argument we could remove the whole `Timestamps` section from `data_types.md` as we do not have any specific "timestamp" datatype. I think it makes sense to somewhere describe the two methods we support for reporting timestamps, but maybe it fits better in the unit section of documentation.

Reply:
Will refactor #717 a bit and then we can remove it here